tests/mgbench/benchmark.py

```diff
@@ -20,63 +20,17 @@
 import multiprocessing
 import random
 import sys
+import statistics
 
 import datasets
 import log
 import helpers
 import runners
 
+
 WITH_FINE_GRAINED_AUTHORIZATION = "with_fine_grained_authorization"
 WITHOUT_FINE_GRAINED_AUTHORIZATION = "without_fine_grained_authorization"
```

Comment:
Regarding this flag, currently on the master running setup is not configurable you were running auth code and non auth code on each run.  Now, you can choose not to run auth code if you do not want. Probably running both should be default?  These strings `"with_fine_grained_authorization"` and `"without_fine_grained_authorization"` are needed because of the graphana and I need them in JSON structure. What I can change  to drop global vars or local vars is is to change a flag to something like this: 
```suggestion
parser.add_argument(
    "--authorization",
    choices=["with_fine_grained_authorization", "without_fine_grained_authorization"],
    default="without_fine_grained_authorization",
    help="Run each query with authorization",
)
```
But this means running just auth or not running auth, not both.

From my perspective, the real questions are:
1. Do we want to run auth as default? Now it pops to me that we should because of graphana. 
2.  Do we need tail latency on auth? Currently, we are not running it.  I think we probably should.

Reply:
Both have to run on CI

Reply:
This is fixed now, both non-auth and auth options are running on CI.